src/APIServer/TokenClientPartial.cs

```diff
@@ -2,6 +2,16 @@
 
 namespace Laserfiche.Api.Client.APIServer
 {
+    /// <summary>
+    /// The Laserfiche Self-Hosted token route API client.
```

Comment:
I am not sure if changing this to the following makes it easier to read, as currently the the sequence is a bit long
The API client for the Laserfiche Self-Hosted token route.

Reply:
I updated the description. thanks